src/mono/netcore/sample/iOS/Program.csproj

```diff
@@ -29,7 +29,7 @@
   <UsingTask TaskName="MonoAOTCompiler"
              AssemblyFile="$(ArtifactsBinDir)MonoAOTCompiler\$(Configuration)\$(NetCoreAppToolCurrent)\MonoAOTCompiler.dll" />
 
-  <Target Name="BuildAppBundle" AfterTargets="CopyFilesToPublishDirectory">
+  <Target Name="BuildAppBundle" AfterTargets="Build">
```

Comment:
Hm we shouldn't be going back to Build, for the libraries tests we do this (i.e. hook the Publish etc targets into AfterTargets=Build):

https://github.com/dotnet/runtime/blob/0fd78a827e0da866174b5a70fd45515da9b7c0a7/eng/testing/tests.mobile.targets#L196-L199

we should do the same here so we can keep using Publish.

Reply:
Reverted to the previous variant; in case of library test build, it will also depend on Publish.